packages/app/src/flame/Flam3.tsx

```diff
@@ -221,7 +216,7 @@ export function Flam3(props: Flam3Props) {
 
     createEffect(() => {
       // redraw when these change
-      const __ = colorGradingPipeline()
+      const _ = colorGradingPipeline()
```

Comment:
Not sure why TS still complains about this, I guess it should be allowed to have ignored variable with that name